cpac_meta_value should always return a string, returns null sometimes

### Description

The `cpac_meta_value` function is used as filter for `ac/column/value` (from the Admin Columns PRo plugin), which always expects its 1st argument to be a string, so the function should always return a string.

Right now, `cpac_meta_value` function sometimes returns `null` when a field has not been initialized. In my case, it's a multiple select field.

Here is the error stack trace:
```
[ERROR] lvl:1 | msg:Uncaught TypeError: ACP\ConditionalFormat\Service\Formatter::format(): Argument #1 ($value) must be of type string, null given, called in .../public_html/wp-includes/class-wp-hook.php on line 308 and defined in .../public_html/wp-content/plugins/admin-columns-pro/classes/ConditionalFormat/Service/Formatter.php:57
Stack trace:
#0 .../public_html/wp-includes/class-wp-hook.php(308): ACP\ConditionalFormat\Service\Formatter->format()
#1 .../public_html/wp-includes/plugin.php(205): WP_Hook->apply_filters()
#2 .../public_html/wp-content/plugins/admin-columns-pro/admin-columns/classes/ListScreen.php(824): apply_filters()
#3 .../public_html/wp-content/plugins/admin-columns-pro/admin-columns/classes/ListScreen/Post.php(63): AC\ListScreen->get_display_value_by_column_name()
#4 .../public_html/wp-includes/class-wp-hook.php(308): AC\ListScreen\Post->manage_value()
#5 .../public_html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#6 .../public_html/wp-includes/plugin.php(517): WP_Hook->do_action()
#7 .../public_html/wp-admin/includes/class-wp-posts-list-table.php(1363): do_action()
#8 .../public_html/wp-admin/includes/class-wp-list-table.php(1532): WP_Posts_List_Table->column_default()
#9 .../public_html/wp-admin/includes/class-wp-posts-list-table.php(1397): WP_List_Table->single_row_columns()
#10 .../public_html/wp-admin/includes/class-wp-posts-list-table.php(807): WP_Posts_List_Table->single_row()
#11 .../public_html/wp-admin/includes/class-wp-posts-list-table.php(783): WP_Posts_List_Table->_display_rows()
#12 .../public_html/wp-admin/includes/class-wp-list-table.php(1444): WP_Posts_List_Table->display_rows()
#13 .../public_html/wp-admin/includes/class-wp-list-table.php(1371): WP_List_Table->display_rows_or_placeholder()
#14 .../public_html/wp-admin/edit.php(487): WP_List_Table->display()
#15 {main}
  thrown | file:.../public_html/wp-content/plugins/admin-columns-pro/classes/ConditionalFormat/Service/Formatter.php | ln:57
```

Here is the Admin Columns Pro format function:
```PHP
// admin-columns-pro/classes/ConditionalFormat/Service/Formatter.php
  /**
   * Comparisons are done case-insensitive
   */
  public function format( string $value, int $id, AC\Column $column ): string {
```

### Version

2.9.9

### Testing Instructions

_No response_

### Screenshots / Screencast

_No response_

### Possible Workaround

None.

### Site Health Information

PHP version: 8.0

### Pods Package

_No response_

--- a/classes/PodsMeta.php
+++ b/classes/PodsMeta.php
@@ -615,8 +615,10 @@ class PodsMeta {
 			$meta = PodsForm::field_method( $pod['fields'][ $field ]['type'], 'ui', $id, $meta, $field, $pod['fields'][ $field ], $pod['fields'], $pod );
 		}
 
-		// always return a string
-		if ( ! is_string( $meta ) ) {
+		// Always return a string version.
+		if ( is_array( $meta ) && isset( $meta[0] ) ) {
+			$meta = pods_serial_comma( $meta, $pod->get_field( $field ) );
+		} elseif ( ! is_string( $meta ) ) {
 			$meta = '';
 		}
 

--- a/classes/PodsMeta.php
+++ b/classes/PodsMeta.php
@@ -529,7 +529,7 @@ class PodsMeta {
 	 * @param int        $id
 	 * @param \AC_Column $obj
 	 *
-	 * @return mixed
+	 * @return string
 	 */
 	public function cpac_meta_value( $meta, $id, $obj ) {
 
@@ -613,6 +613,11 @@ class PodsMeta {
 			}
 
 			$meta = PodsForm::field_method( $pod['fields'][ $field ]['type'], 'ui', $id, $meta, $field, $pod['fields'][ $field ], $pod['fields'], $pod );
+		}
+
+		// always return a string
+		if ( ! is_string( $meta ) ) {
+			$meta = '';
 		}
 
 		return $meta;

